### ISSUE TITLE
Is performance of JSONObject.get() and JSONObject[] method slower than python dict?

### ISSUE THREAD

**Issue by gzrjzcx** [TeskaLabs/cysimdjson#52]
In my case, the speed of parser.parse() is about 4x faster than orjson.loads(), great!
However, both JSONObject.get() and [] method to access inner attribute is 2x slower than python dict.get() and [] mthod. 

Is there any good ideas to load json use this module, and use python dict to access value? ps., export() method is too heavily.
Or is there any other more efficient method to access the value?

Thanks

**Comment by ghost** [comment]
Have you found anything?

**Comment by gzrjzcx** [comment]
> Have you found anything?

No, finally I dont use this method. If u have any alternative approach, please tell me. Thanks.
